### DISCUSSION TITLE
Emphesize the need for Sponsor?

### DISCUSSION THREAD

**Discussion by pskowronek** [mucommander/mucommander#1008] [General]
Update both muCommander website and README.md to emphasize the need for sponsorship to be able sign muC officially on macOS (or any other platform that might require that now or in the future)?

![obraz](https://github.com/mucommander/mucommander/assets/360665/69eb2ae3-f15c-4c47-a01f-d53fda580bca)

The popularity of muC on macOS (especially on mac silicon due to more security restrictions) may drop drastically and the work to handle raised bugs related to it  will be quite big. Like here: https://github.com/mucommander/mucommander/issues/1005

**Comment by ahadas** [comment]
Hi @pskowronek , thanks for raising this. Yeah, that was the plan - just recently I got a notification from the other service we used for donations, Flattr, that they change their model and stop supporting micro-donation, and therefore set up GitHub sponsership for muCommander. The transition was completed this week and I didn't want want to publish it too much before seeing that GitHub sponsership works well for muCommander. I think it's time not only to replace Flattr with GitHub sponsership (already done on the website) but also indicate the great contributions we have been seeing recently and that if users would like to support this, and signing the dmg (and as you wrote, possibly other packages), they should consider making a donation. We used to get contributions via Flattr that covered things like service for building the nightly release (which was replaced with github hosted runners) and the developer account I used in the past - if we get similar donations, it would cover all our expenses. I'll tweet about it from the muCommander account

**Comment by pskowronek** [comment]
@ahadas I've just updated Thunderbird and it opened the following page: https://www.thunderbird.net/en-US/thunderbird/115.0/appeal/
Maybe muC should do something akin? Like display what are the new features plus a link to Donation on github?

**Comment by pskowronek** [comment]
@ahadas - are you able to check how much money muC has received? I think muC should collect at least two-year worth of apple subscription fee to start thinking on signing DMG.

**Reply by ahadas** [reply]
yeah, we got 80$ so far
about waiting for collecting at least two-year worth of Apple subscription fee, it would be great to reach that point but as I didn't see we get a discount when purchasing two-year deal, even one-year deal would be ok

**Reply by pskowronek** [reply]
I mean, to have longer term period of signing DMGs, not like in just 1 year once again DMG are unsigned.
